src/Sarif/Schemata/sarif-schema.json

```diff
@@ -521,6 +521,17 @@
           }
         },
 
+        "threadFlowLocations": {
+          "description": "An array of threadFlowLocation objects cached in external file.",
```

Comment:
>"An array of threadFlowLocation objects cached in external file." [](start = 25, length = 65)

please help improve this (and other descriptions), we also need to update the not-yet-autogen files if applicable.